test/Microsoft.Azure.WebJobs.Host.UnitTests/Scale/ProcessMonitorTests.cs

```diff
@@ -51,6 +52,62 @@ public ProcessMonitorTests()
                 }
             };
 
+        [Fact]
+        public void Process_TotalProcessorTime_ReturnsExpectedResult()
+        {
+            var process = Process.GetCurrentProcess();
+            var monitor = new ProcessMonitor(process);
+
+            Assert.Equal(monitor.Process.TotalProcessorTime, process.TotalProcessorTime);
+            Assert.Equal(process.TotalProcessorTime, process.TotalProcessorTime);
+
+            process.Refresh();
+
+            Assert.Equal(process.TotalProcessorTime, process.TotalProcessorTime);
+        }
+
+        [Fact]
+        public void Process_PrivateMemoryBytes_ReturnsExpectedResult()
+        {
+            var process = Process.GetCurrentProcess();
+            var monitor = new ProcessMonitor(process);
+
+            Assert.Equal(monitor.Process.PrivateMemoryBytes, process.PrivateMemorySize64);
+            Assert.Equal(monitor.Process.PrivateMemoryBytes, process.PrivateMemorySize64);
```

Comment:
Whats the point of doing this twice?

Reply:
Removed the extra line here and in the other tests.